cli/src/commands/start.rs

```diff
@@ -566,7 +583,7 @@ fn load_or_compute_genesis<N: Network>(
     /* Otherwise, compute the genesis block and store it. */
 
     // Initialize a new VM.
-    let vm = VM::from(ConsensusStore::<N, ConsensusMemory<N>>::open(None)?)?;
+    let vm = VM::from(ConsensusStore::<N, ConsensusMemory<N>>::open(storage_mode)?)?;
```

Comment:
Is passing `storage_mode` through necessary here? We are using ConsensusMemory, so any StorageMode will have the same result.

Reply:
Resolved [2b1b881](https://github.com/AleoHQ/snarkOS/pull/3044/commits/2b1b881e9ec9bd5839cdab33f45ce744f70818a6)